### DISCUSSION TITLE
Possible to view the value in hex instead of decimal?

### DISCUSSION THREAD

**Discussion by pepeu93** [zero-plusplus/vscode-autohotkey-debug#187] [Q&A] [answered]
I was reading here:
https://github.com/zero-plusplus/vscode-autohotkey-debug/wiki/Data-inspection

Couldn't find so I'm asking, is possible to view the value when hovering a variable in hex instead of decimal?
Or just copy as hex?

**Comment by zero-plusplus** [comment]
It is not possible to display hex number in the hover information.

Although it is technically possible to display hex, I don't plan to implement it because I have decided that the hover information should match the value displayed in the data inspection, and I have also decided that the data inspection should display the actual value of AutoHotkey.

However, the same request has been made before, and to get as close to the request as possible, I implemented the context menu you saw.

So please use `View as Hex` or `Copy as Hex`

**Reply by pepeu93** [reply]
I did not understand how to enable the `View as Hex`

**Comment by zero-plusplus** [comment, marked as answer]
Sorry. It's `View as Each Base Numbers`, not `View as Hex`.

This will show all supported formats, including hex.

I forgot that I changed it to reduce the number of context menus.
